src/style.css

```diff
@@ -0,0 +1,35 @@
+body {
+    background-color: #f9eae1;
+  }
+
+button {
+  margin: 5px 2px 5px 2px;
+  background-color: #cc8b86;
+}
+
+#messageContainer {
+  display: flex;
+  justify-content: flex-end;
+}
+
+#yourMessage {
+  background-color: #7d4f50;
+  color: white;
+  border-radius: 7px;
+  padding: 5px;
+  margin: 2px;
+  text-align: right;
+}
+
+#author {
+  font-size: 10px;
+}
+
+#previousMessage {
+  background-color: #bb9d99;
+  color: white;
```

Comment:
It is better to be consistent about the color format, you can use white as `#fff`